Teacher-Training.md

```diff
@@ -7,9 +7,7 @@ Teacher training generally covers two areas:
 
 ## Being a Good Teacher
 
-We currently rely on the [RailsBridge teacher training](http://docs.railsbridge.org/workshop/more_teacher_training) slide deck.
-
-_We'd love a volunteer to adapt this into something specific to ClojureBridge - hint, hint!_
+See [teacher training](http://clojurebridge.github.io/curriculum/teacher-training.html)
```

Comment:
Oh heyyyyy it totally will work, at least in @KatieStarks's fork, when that other PR is merged. Okay! Carry on!